src/test/abi_component/widget.cpp

```diff
@@ -70,11 +70,11 @@ struct widget_factory : iwidget_factory
         else
         {
             *object = nullptr;
-            return xlang_hresult_no_interface;
+            return com_interop_result::no_interface;
         }
 
         AddRef();
-        return 0;
+        return com_interop_result::success;
```

Comment:
This wouldn't be necessary if the return type was simply int32_t. Seems overly verbose for an ABI.